### PR TITLE
Container fix

### PR DESCRIPTION
**Pull request by gerhard-tinned**
Fixes issues find while deploying nictool v2.

Changes proposed in this pull request:
- missing nt_vars export of DB_ENGINE
- Additional debug output in startup.sh

### CODE REVIEW

File: dist/docker/debian/web/startup.sh [hunk 1]
```diff
@@ -4,77 +4,88 @@
...
 fi
 if [ ! -L "/etc/apache2/sites-enabled/nictool.conf" ]; then 
-	ln -s -T ${CONF_DIR}/nictool.conf /etc/apache2/sites-enabled/nictool.conf 
+	ln -v -s -T ${CONF_DIR}/nictool.conf /etc/apache2/sites-enabled/nictool.conf 
 fi
 
-# Lead the nt_vars environment variables
+
+echo ""
+echo -e "***\n*** Lead the nt_vars environment variables\n***\n"
```
Comment (msimerson) [inline dist/docker/debian/web/startup.sh:40]: typo: Lead -> Load

**Review by msimerson — APPROVED** [review]
Looks good, I'll merge after the typo is corrected.

File: dist/docker/debian/web/startup.sh (hunk 1 above)
Comment (msimerson) [reply dist/docker/debian/web/startup.sh:40]: still waiting on that typo fix.

**Comment by gerhard-tinned** [comment]
I fixed the typo ... thanks for spotting it.

**Pull request merged**